.zuul.d/config.yaml

```diff
@@ -3,47 +3,137 @@
     name: SovereignCloudStack/security-infra-scan-pipeline
     default-branch: main
     merge-mode: "squash-merge"
-    periodic-daily:
+    check:
       jobs:
-        - scs-daily-security-scan
-    periodic-weekly:
+        - scs-baseline-security-scan-test
+        - scs-full-security-scan-test
+        - scs-greenbone-security-scan-test
+    gate:
       jobs:
-        - scs-weekly-security-scan
-    post:
+        - scs-baseline-security-scan-test
+        - scs-full-security-scan-test
+        - scs-greenbone-security-scan-test
+    periodic-daily:
       jobs:
-        - noop
-    check:
+        - scs-baseline-security-scan
+    periodic-weekly:
       jobs:
-        - noop
+        - scs-full-security-scan
+        - scs-greenbone-security-scan
 
 - job:
-    name: scs-daily-security-scan
+    name: scs-security-scan-base
     parent: base
-    timeout: 7200 #2h
+    description: |
+      Base for any security scanning job
+    timeout: 3600 #1h
+    pre-run: playbooks/pre.yaml
+    post-run: playbooks/post.yaml
+    vars:
+      # Where to store intermediate data
+      scan_work_dir: "{{ ansible_user_dir }}/wrk"
+      # Where to flush (and read) results from
+      scan_results_dir: "{{ ansible_user_dir }}/wrk/scan_results"
+      # Default list of scan targets.
+      # NOTE(gtema): It would be nice in tests to use test instance itself, but
+      # it only exposes SSH port what makes scanning very fast and reports
+      # empty. We potentially want to see how the reports are looking like also 
+      # in tests.
+      scan_targets:
+        - 213.131.230.161
```

Comment:
I'm not sure if we want to hardcode targets here. Maybe neither `target.txt` nor `scan_targets` let us reach our goal of dynamically passing a list of hosts into the job?

Reply:
idea of defining it on the job level was to make parameterization possible at all (compared to hardcoding it into the mounted file). Once there is a way to determine dynamically target this can be used. So far there is no known to me way.